### PR TITLE
Add input and output

### PR DESCRIPTION
**Pull request by steinbrueckri**
# Description

Add the ability to set mdl path and use the output from mdl.

## Check List

- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by Xorima** [comment]
Hey @steinbrueckri 
thanks for this, I will get it reviewed and tested later today and merge it

File: entrypoint.sh [hunk 1]
```diff
@@ -1 +1,7 @@
-mdl ./
\ No newline at end of file
+output=$(mdl $INPUT_PATH)
+echo ::set-output name=output::"$output"
+if [ ! -z "$output" ]
+then
+    echo "FOUND MARKDOWN ISSUS"
```
Comment (rshade) [inline entrypoint.sh]: Will it output the actual errors still? Can you paste an example output here?

**Review by rshade — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (steinbrueckri) [reply entrypoint.sh]: https://github.com/steinbrueckri/gha-test/runs/430139684

**Review by rshade — APPROVED** [review]

**Comment by rshade** [comment]
@steinbrueckri Awesome thanks man. I have approved, not sure if @Xorima wants me to merge. I will check.

**Comment by Xorima** [comment]
merged

**Pull request merged**